projects/plugins/jetpack/class.jetpack.php

```diff
@@ -4964,18 +4967,18 @@ public function alert_auto_ssl_fail() {
 
 		<div id="jetpack-ssl-warning" class="error jp-identity-crisis">
 			<div class="jp-banner__content">
-				<h2><?php _e( 'Outbound HTTPS not working', 'jetpack' ); ?></h2>
-				<p><?php _e( 'Your site could not connect to WordPress.com via HTTPS. This could be due to any number of reasons, including faulty SSL certificates, misconfigured or missing SSL libraries, or network issues.', 'jetpack' ); ?></p>
+				<h2><?php esc_html_e( 'Outbound HTTPS not working', 'jetpack' ); ?></h2>
+				<p><?php esc_html_e( 'Your site could not connect to WordPress.com via HTTPS. This could be due to any number of reasons, including faulty SSL certificates, misconfigured or missing SSL libraries, or network issues.', 'jetpack' ); ?></p>
 				<p>
-					<?php _e( 'Jetpack will re-test for HTTPS support once a day, but you can click here to try again immediately: ', 'jetpack' ); ?>
-					<a href="#" id="jetpack-recheck-ssl-button"><?php _e( 'Try again', 'jetpack' ); ?></a>
-					<span id="jetpack-recheck-ssl-output"><?php echo get_transient( 'jetpack_https_test_message' ); ?></span>
+					<?php esc_html_e( 'Jetpack will re-test for HTTPS support once a day, but you can click here to try again immediately: ', 'jetpack' ); ?>
+					<a href="#" id="jetpack-recheck-ssl-button"><?php esc_html_e( 'Try again', 'jetpack' ); ?></a>
+					<span id="jetpack-recheck-ssl-output"><?php echo get_transient( 'jetpack_https_test_message' ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- not sure what this is returning and if escaping will be weird. ?></span>
```

Comment:
It gets set just above, on line 4951. Looks like `esc_html()` would be a good idea, since it might include arbitrary text of a network error response.

Reply:
4e8c9ca5db7b806f2f7b8e146ea95db47214fbe7